src/models/ClientEnv.ts

```diff
@@ -0,0 +1,4 @@
+export enum ClientEnv {
+  Production,
+  Staging,
+}
```

Comment:
Local isn't listed here because if you are using the sdk/frontend against a local env, you will need to inject your own Config anyways, so it doesn't matter.

Reply:
We should prob add support for this via env vars later. I mentioned this somewhere else too